src/components/qas/questions/qaFillableTable/FillableTableRangeInput.jsx

```diff
@@ -136,7 +136,7 @@ class FillableTableRangeInput extends React.PureComponent {
     const answered = !isEmpty(answer);
     return (
       <div className="table-cell-range-input">
-        {qaReview && <StellarValueRange data={answer.data} />}
+        {qaReview && <StellarValueRange data={answered ? answer.data : []} />}
```

Comment:
use `answered` to pass either the answer data or an empty array.

Reply:
good catch